src/polar/RadialBar.tsx

```diff
@@ -416,83 +536,27 @@ export class RadialBar extends PureComponent<RadialBarProps> {
     const stackedDomain = stackedData ? numericAxis.scale.domain() : null;
     const baseValue = getBaseValueOfBar({ numericAxis });
     const cells = findAllByType(children, Cell);
-    const sectors = displayedData.map((entry: any, index: number) => {
-      let value, innerRadius, outerRadius, startAngle, endAngle, backgroundSector;
-
-      if (stackedData) {
-        value = truncateByDomain(stackedData[dataStartIndex + index], stackedDomain);
-      } else {
-        value = getValueByDataKey(entry, dataKey);
-        if (!Array.isArray(value)) {
-          value = [baseValue, value];
-        }
-      }
-
-      if (layout === 'radial') {
-        innerRadius = getCateCoordinateOfBar({
-          axis: radiusAxis,
-          ticks: radiusAxisTicks,
-          bandSize,
-          offset: pos.offset,
-          entry,
-          index,
-        });
-        endAngle = angleAxis.scale(value[1]);
-        startAngle = angleAxis.scale(value[0]);
-        outerRadius = innerRadius + pos.size;
-        const deltaAngle = endAngle - startAngle;
-
-        if (Math.abs(minPointSize) > 0 && Math.abs(deltaAngle) < Math.abs(minPointSize)) {
-          const delta = mathSign(deltaAngle || minPointSize) * (Math.abs(minPointSize) - Math.abs(deltaAngle));
-
-          endAngle += delta;
-        }
-        backgroundSector = {
-          background: {
-            cx,
-            cy,
-            innerRadius,
-            outerRadius,
-            startAngle: props.startAngle,
-            endAngle: props.endAngle,
-          },
-        };
-      } else {
-        innerRadius = radiusAxis.scale(value[0]);
-        outerRadius = radiusAxis.scale(value[1]);
-        startAngle = getCateCoordinateOfBar({
-          axis: angleAxis,
-          ticks: angleAxisTicks,
-          bandSize,
-          offset: pos.offset,
-          entry,
-          index,
-        });
-        endAngle = startAngle + pos.size;
-        const deltaRadius = outerRadius - innerRadius;
-
-        if (Math.abs(minPointSize) > 0 && Math.abs(deltaRadius) < Math.abs(minPointSize)) {
-          const delta = mathSign(deltaRadius || minPointSize) * (Math.abs(minPointSize) - Math.abs(deltaRadius));
-          outerRadius += delta;
-        }
-      }
-
-      return {
-        ...entry,
-        ...backgroundSector,
-        payload: entry,
-        value: stackedData ? value : value[1],
-        cx,
-        cy,
-        innerRadius,
-        outerRadius,
-        startAngle,
-        endAngle,
-        ...(cells && cells[index] && cells[index].props),
-        // @ts-expect-error missing types
-        tooltipPayload: [getTooltipItem(item, entry)],
```

Comment:
Hmmmm I did not include the tooltip payload and position. Do we have RadialBar tooltip coming from redux already?

Reply:
I think yes? lol

Tooltip is still defaulted to context and falls back on redux though

Reply:
Works fine in my storybook

<img width="475" alt="image" src="https://github.com/user-attachments/assets/fdb65439-badd-4a9a-aaa9-c5a10a65a726">

Reply:
I don't think it was ever dependent on this particular line. This was just so the tooltip data for each item is included in each item

Reply:
Visual diff https://www.chromatic.com/build?appId=63da8268a0da9970db6992aa&number=1518

Reply:
Yep lgtm